lib/ansible/modules/packaging/os/homebrew_bundle.py

```diff
@@ -0,0 +1,510 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2018, Daniel Jaouen <dcj24@cornell.edu>
+#
+# Based on macports (Jimmy Tang <jcftang@gmail.com>)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+RETURN = ''' # '''
+
+DOCUMENTATION = '''
+---
+module: homebrew_bundle
+author:
+    - "Daniel Jaouen (@danieljaouen)"
+requirements:
+   - "python >= 2.6"
+short_description: Manager for Homebrew bundles
+description:
+    - Manages Homebrew bundles
+version_added: "2.8"
+options:
+    state:
+        description:
+            - the state of the homebrew bundle
+        choices: ['installed', 'dumped', 'cleanup']
+        default: installed
+    file_path:
+        description:
+            - The path of the Brewfile. Defaults to ~/.Brewfile.
+        required: false
+        default: ~/.Brewfile
+    path:
+        description:
+            - "A ':' separated list of paths to search for 'brew' executable.
+              Since a package (I(formula) in homebrew parlance) location is prefixed relative to the actual path of I(brew) command,
```

Comment:
```suggestion
            - Since a package (I(formula) in homebrew parlance) location is prefixed relative to the actual path of I(brew) command,
```